lib/v2/components/dashboard/components/currency_info_card.dart

```diff
@@ -0,0 +1,87 @@
+
+
+import 'package:flutter/cupertino.dart';
```

Comment:
What does this entail? Put it in a different folder? A different package?

Not sure what the actionable is here! @gguijarro-c-chwy

Reply:
![image](https://user-images.githubusercontent.com/8041901/120131841-ccc65980-c18e-11eb-977e-6550be79f913.png)
The dashboard is a screen
This `dashboard` screen is composed by multiple components like the one you are working on now. 

We need a folder under v2 screens for dashboard
And inside this folder you will have a `components` folder. 
This code goes inside that components folder. 

Since this widget belongs to the dashboard alone and its not shared in other places of the app . 

Right now, its placed in the global components folder.

Reply:
Ah ok got it, thanks!

Reply:
Done - unfortunately git doesn't understand I'm just moving files. Sigh.